host/volume/zfs/integrations.go

```diff
@@ -0,0 +1,32 @@
+package zfs
+
+import (
+	"fmt"
+	"strings"
+
+	gzfs "github.com/flynn/flynn/Godeps/_workspace/src/github.com/mistifyio/go-zfs"
+)
+
+type Logger struct{}
+
+func (*Logger) Log(msg []string) {
+	fmt.Printf("[zfs] %s\n", strings.Join(msg, " ")) // TODO replace with log15
```

Comment:
I'd just swap this out for context-less Debug-level calls with a `package=zfs` or something annotation.